codersdk/deployment.go

```diff
@@ -2147,6 +2148,18 @@ when required by your organization's security policy.`,
 			Group:       &deploymentGroupIntrospectionLogging,
 			YAML:        "enableTerraformDebugMode",
 		},
+		{
+			Name: "Additional CSP Policy",
+			Description: "Coder configures a Content Security Policy (CSP) to protect against XSS attacks. " +
+				"This setting allows you to add additional CSP directives, which can open the attack surface of the deployment. " +
+				"Format matches the CSP directive format, e.g. --additional-csp-policy=\"script-src https://example.com\".",
```

Comment:
suggestion: add a link to https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP or something

Reply:
Maybe our flags should have a `DocsLink: "https://..."` field.

Reply:
I feel the description is enough as is. The link and wording around it makes this description so long.